packages/kbn-search-index-documents/components/document_list.tsx

```diff
@@ -53,6 +57,10 @@ export const DocumentList: React.FC<DocumentListProps> = ({
   meta,
   onPaginate,
   setDocsPerPage,
+  defaultVisibleFields = 3,
```

Comment:
suggestion - have a `resultProps` prop which passes down these to the Result component?